lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -79,82 +80,57 @@ def expects_uri?
           term_config.term_id_expects_uri?
         end
 
-        def extract_uri_from_id(id)
-          return @uri = id if expects_uri?
+        def uri
+          return @uri if @uri.present?
+          return @uri = RDF::URI.new(id) if expects_uri?
```

Comment:
Fix bug -- Exception was thrown when `expects_uri? # true` because `@uri` was getting set to a `String` when it needs to be an `RDF::URI`.